.github/workflows/ci.yml

```
@@ -153,6 +153,53 @@ jobs:
 
     - run: make rust-benchmark
 
+  emscripten:
+    name: build on emscripten
```

```suggestion
  build_wasm:
    name: build wasm
```

?? will make more sense to most people I guess.

---

For the time being maybe, but there are also `wasm32-unknown-unknown` and `wasm32-wasi` targets which may gain support in the future.

---

then `wasm_ emscripten`?

It doesn't really matter about changing the name in future when more targets come along.